config.yml

```diff
@@ -648,6 +678,135 @@ production_factories:
     repair_inputs:
       Diamond:
         material: DIAMOND
+  Emerald_Cauldron:
+    name: Emerald Cauldron
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:  
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 192   
+      Diamonds:
+        material: DIAMOND
+        amount: 128   
+      Iron:
+        material: IRON_INGOT
+        amount: 256   
+    recipes:
+      - Emerald_XP_0
+      - Emerald_XP_1
+      - Emerald_XP_2
+    repair_multiple: 2
+    repair_inputs:
+      Emeralds:
+        material: EMERALD
+        amount: 32   

```

Comment:
Convention check: 10% of install would be approx 19 Emeralds, vs. 32 listed here.